protocol/communities/community.go

```diff
@@ -1820,7 +1821,20 @@ func (o *Community) RequestedToJoinAt() uint64 {
 }
 
 func (o *Community) nextClock() uint64 {
-	return o.config.CommunityDescription.Clock + 1
+	if o.timesource == nil {
+		return o.config.CommunityDescription.Clock + 1
```

Comment:
when would this be the case, only testing? maybe otherwise it would have been best to throw an error, as it might mean something is misconfigured (though is likely fine either ways)

Reply:
good point :+1:  addressed, please re-check